salt/config/__init__.py

```diff
@@ -43,8 +43,9 @@
 
 try:
     import psutil
+    psutil.virtual_memory
```

Comment:
I think it would be cleaner to use `hasattr()` here instead of this try/except block.